tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html

```diff
@@ -30,7 +30,7 @@
 <link rel="import" href="../tf-dashboard-common/dashboard-style.html">
 <link rel="import" href="../tf-dashboard-common/tf-dashboard-layout.html">
 <link rel="import" href="../tf-dashboard-common/tf-option-selector.html">
-<link rel="import" href="../tf-data-selector/tf-data-selector.html">
+<link rel="import" href="../tf-data-selector/type.html">
```

Comment:
Shouldn't it still be importing tf-data-selector.html?  I thought type.html was just an HTML import placeholder for type.js which actually will probably be empty (since a .ts with only type definitions doesn't really map to any JS code), but certainly won't include the `tf-data-selector` element?

Same thing for the BUILD dep on //tensorboard/components/tf_data_selector

Reply:
@nfelt probably true for type only file but it does not seem to be the case especially when you define enums (kinda obvious)

from bazel-genfiles:
```
/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.                                                                                                                       

Licensed under the Apache License, Version 2.0 (the 'License');
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an 'AS IS' BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
==============================================================================*/
var tf_data_selector;
(function (tf_data_selector) {
    var Type;
    (function (Type) {
        Type[Type["WITHOUT_EXPERIMENT"] = 0] = "WITHOUT_EXPERIMENT";
        Type[Type["SINGLE"] = 1] = "SINGLE";
        Type[Type["COMPARISON"] = 2] = "COMPARISON";
    })(Type = tf_data_selector.Type || (tf_data_selector.Type = {}));
})(tf_data_selector || (tf_data_selector = {})); // namespace tf_data_selector
```

Reply:
Oh, right yes, I was forgetting tf-data-selector isn't imported into the individual dashboards.  And right, ok, an enum would require JS code, I was thinking of tf_backend type.ts which does compile down to an empty file.

Reply:
:D thanks for confirming